data/tabular/qm9/meta.yaml

```diff
@@ -0,0 +1,187 @@
+    - id: gap
+      description: homo lumogap, difference between LUMO and HOMO
+      units: Hartree
+      type: float
+      names:
+          - noun: gap
+          - noun: homo lumo gap
+    - id: r2
+      description: electronic spatial extent
+      units: Bohr^2
+      type: float
+      names:
+          - noun: r2
+          - noun: electronic spatial extent
+    - id: zero_point_energy
+      description: zero point vibrational energy
+      units: Hartree
+      type: float
+      names:
+          - noun: zero point vibrational energy
+          - noun: zero point energy
+          - noun: zpve
+    - id: u0
+      description: internal energy at 0 K
+      units: Hartree
+      type: float
+      names:
+          - noun: internal energy at 0 K
+    - id: u298
+      description: internal energy at 298.15 K
+      units: Hartree
+      type: float
+      names:
+          - noun: internal energy at 298.15 K
+    - id: h298
+      description: enthalpy at 298.15 K
+      units: Hartree
+      type: float
+      names:
+          - noun: enthalpy at 298.15 K
+    - id: g298
+      description: free energy at 298.15 K
+      units: Hartree
+      type: float
+      names:
+          - noun: free energy at 298.15 K
+          - noun: gibbs free energy at 298.15 K
+    - id: heat_capacity
+      description: heat capacity at 298.15 K
+      units: cal/(mol K)
+      type: float
+      names:
+          - noun: heat capacity at 298.15 K
+identifiers:
+    - id: smiles
+      type: SMILES
+      description: SMILES
+    - id: inchi
+      type: InChI
+      description: International Chemical Identifier
+license: CC0 1.0
+num_points: 133885
+links:
+    - url: https://figshare.com/collections/Quantum_chemistry_structures_and_properties_of_134_kilo_molecules/978904
+      description: original data source
+    - url: https://huggingface.co/datasets/n0w0f/qm9-csv/blob/main/qm9_dataset.csv
+      description: parsed dataset in csv format
+bibtex:
+    - |-
+      @article{ramakrishnan2014quantum,
+      title={Quantum chemistry structures and properties of 134 kilo molecules},
+      author={Ramakrishnan, Raghunathan and Dral, Pavlo O and Rupp, Matthias and Von Lilienfeld, O Anatole},
+      journal={Scientific data},
+      volume={1},
+      number={1},
+      pages={1--7},
+      year={2014},
+      publisher={Nature Publishing Group}}
+    - |-
+      @article{ruddigkeit2012enumeration,
+      title={Enumeration of 166 billion organic small molecules in the chemical universe database GDB-17},
+      author={Ruddigkeit, Lars and Van Deursen, Ruud and Blum, Lorenz C and Reymond, Jean-Louis},
+      journal={Journal of chemical information and modeling},
+      volume={52},
+      number={11},
+      pages={2864--2875},
+      year={2012},
+      publisher={ACS Publications}}
+templates:
+    - The molecule with the {smiles__description} representation of {smiles#} has a dipole moment of {dipole_moment#} Debye.
```

Comment:
That's very cool! Thanks for adding! 👍🏽 💯 Awesome that you also got started with the templates without any onboarding!
I'll review it tomorrow in more detail.  

Some first comments:
- Do you think we need more details about the computational methods in the prompts? And do you think we can come up with a reasonable number of digits to round to? 
- Perhaps you can also use some sampling for `the molecule` to increase the diversity? 
- The replacement of SMILES with InChI etc. does anyhow already happen in the sampling, so no need to include it in the templates (but this is not (very well/at all) documented)

Reply:
Hi Kevin, Thank you for the review .
1. I think it is a good idea to add computational details. But to what extend should we include the details, should it be just mentioning the method and XC (eg: calculated using DFT at the level of B3LYP accuracy) or should we include more details ? (for eg: post processing techniques pertaining to the derived properties ? + pseudo potentials + etc )
2.  sorry,  Can you elaborate on the second point ? are you saying we should shuffle the dataset or while generating templates there should be a helper function to sample from the dataset ?
3. understood.

Reply:
Sorry for the delay in coming back to this

1. The functional should be good enough for now. I do not expect that we can resolve more than that at the moment. 
2. Sorry for being unclear. We do not want to overfit to specific sentences. If you could sample `{|}` over different synonyms for "molecule", e.g. "compound", "chemical", ... it would help a lot

Thanks for the contribution!